tool/gravity/cli/install.go

```diff
@@ -743,6 +746,29 @@ func completeJoinPlan(localEnv, joinEnv *localenv.LocalEnvironment, operation *o
 	return joinFSM.Complete(trace.Errorf("completed manually"))
 }
 
+func completeJoinPlanFromAnotherNode(localEnv *localenv.LocalEnvironment, operation *ops.SiteOperation) error {
+	clusterEnv, err := localEnv.NewClusterEnvironment()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if operation == nil {
+		operation, err = ops.GetExpandOperation(clusterEnv.Backend)
```

Comment:
Which one is it going to return? Just wondering what might happen if multiple expands are running. Maybe always require --operation-id when completing expands?